identifiers/country-cz/README.md

```diff
@@ -0,0 +1,8 @@
+# OCD Division Identifier
+
+This folder list the following administrative divisions of the Czech Republic :
+
+*  Electoral districts [ref](https://www.senat.cz/senat/volby/hledani/obvody.php?ke_dni=30.1.2020&O=12) as **ed**.
```

Comment:
Might be good to clarify here that the electoral districts are for the Senate.

Reply:
Done.